pkg/cmd/cli/backup/create.go

```diff
@@ -154,13 +172,18 @@ func (o *CreateOptions) Complete(args []string, f client.Factory) error {
 }
 
 func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
-	backup := &api.Backup{
-		ObjectMeta: metav1.ObjectMeta{
-			Namespace: f.Namespace(),
-			Name:      o.Name,
-			Labels:    o.Labels.Data(),
-		},
-		Spec: api.BackupSpec{
+	var backup *api.Backup
+	backupBuilder := builder.ForBackup(f.Namespace(), o.Name).ObjectMeta(builder.WithLabelsMap(o.Labels.Data()))
+
+	if o.FromSchedule != "" {
+		schedule, err := o.client.VeleroV1().Schedules(f.Namespace()).Get(o.FromSchedule, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		backup = backupBuilder.FromSchedule(schedule).Result()
```

Comment:
If I'm reading this right, any labels duplicated between the ones provided by the user and on the schedule will be set to the values on the schedule, correct?

If this is what we want, ti should be clearly documented on the `--from-schedule` argument. However, I think in general values provided at the command line usually win over things like config files or the schedule's values.

Reply:
Ah yes, good catch - I agree that the labels provided on the command line should take precedence here. Will change the ordering here.